projects/w11k/ngx-select/src/lib/core/ngx-select.core.spec.ts

```diff
@@ -197,7 +206,7 @@ describe('NgxSelect', () => {
       expect(result.length).toBe(9);
     });
 
-    it('should filter options with last string with new options', async(() => {
+    it('should filter options with last string with new options', () => {
       const filterString = '1';
 
       const options = [
```

Comment:
`async()` function is deprecated and should be replaced with `waitForAsync()`. Only dropping could result in false positive tests. As an alternative we could use async await.